Refresh ubuntu container images

jovial opened the pull request:
Periodic refresh to get updated packages.

Build logs: https://github.com/stackhpc/stackhpc-kayobe-config/actions/runs/4184397411

jovial committed:
Refresh ubuntu containers

Periodic refresh to get updated packages.

Build logs: https://github.com/stackhpc/stackhpc-kayobe-config/actions/runs/4184397411

markgoddard reviewed (changes requested):
Needs a release note

jovial committed:
Adds release note for ubuntu container refresh

Package versions from output below.

old:

```javascript
()[root@01b9e3ddbfbb /]# dpkg -l | grep libvirt
ii  libvirt-clients                      6.0.0-0ubuntu8.16                     amd64        Programs for the libvirt library
ii  libvirt-daemon                       6.0.0-0ubuntu8.16                     amd64        Virtualization daemon
ii  libvirt-daemon-driver-qemu           6.0.0-0ubuntu8.16                     amd64        Virtualization daemon QEMU connection driver
ii  libvirt-daemon-system                6.0.0-0ubuntu8.16                     amd64        Libvirt daemon configuration files
ii  libvirt-daemon-system-systemd        6.0.0-0ubuntu8.16                     amd64        Libvirt daemon configuration files (systemd)
ii  libvirt0:amd64                       6.0.0-0ubuntu8.16                     amd64        library for interfacing with different virtualization systems
```

new:

```javascript
()[root@3493c5fdc96a /]# dpkg -l | grep libvirt
ii  libvirt-clients                      6.0.0-0ubuntu8.16                     amd64        Programs for the libvirt library
ii  libvirt-daemon                       6.0.0-0ubuntu8.16                     amd64        Virtualization daemon
ii  libvirt-daemon-driver-qemu           6.0.0-0ubuntu8.16                     amd64        Virtualization daemon QEMU connection driver
ii  libvirt-daemon-system                6.0.0-0ubuntu8.16                     amd64        Libvirt daemon configuration files
ii  libvirt-daemon-system-systemd        6.0.0-0ubuntu8.16                     amd64        Libvirt daemon configuration files (systemd)
ii  libvirt0:amd64                       6.0.0-0ubuntu8.16                     amd64        library for interfacing with different virtualization systems
```

old:

```
(venv-pulp) [cloud-user@will-control-host-0 ~]$ docker run -it pulp-server.internal.sms-cloud:8080/stackhpc-dev/ubuntu-source-openvswitch-vswitchd:wallaby-20221222T161624 bash
()[root@9f15e78c72fc /]# ovs-vswitchd --version
ovs-vswitchd (Open vSwitch) 2.17.2
()[root@9f15e78c72fc /]# dpkg -l | grep openvswitch-switch
ii  openvswitch-switch             2.17.2-0ubuntu0.22.04.2~cloud0    amd64        Open vSwitch switch implementations
```

new:

```
()[root@ca7be869ba89 /]# dpkg -l | grep vswitch
ii  openvswitch-common             2.17.3-0ubuntu0.22.04.1~cloud0    amd64        Open vSwitch common components
ii  openvswitch-switch             2.17.3-0ubuntu0.22.04.1~cloud0    amd64        Open vSwitch switch implementations
ii  python3-openvswitch            2.17.3-0ubuntu0.22.04.1~cloud0    all          Python 3 bindings for Open vSwitch
```

old:

```
()[root@c2f9a61b303e /]# dpkg -l | grep ovn
ii  ovn-central                    22.03.0-0ubuntu1~cloud0           amd64        OVN central components
ii  ovn-common                     22.03.0-0ubuntu1~cloud0           amd64        OVN common components
```

new:

```
Status: Downloaded newer image for pulp-server.internal.sms-cloud:8080/stackhpc-dev/ubuntu-source-ovn-northd:wallaby-20230215T134157
()[root@fd78907feea3 /]# dpkg -l | grep ovn
ii  ovn-central                    22.03.0-0ubuntu1~cloud0           amd64        OVN central components
ii  ovn-common                     22.03.0-0ubuntu1~cloud0           amd64        OVN common components
```

jovial commented:
Promoting overcloud images: https://github.com/stackhpc/stackhpc-release-train/actions/runs/4224699230
Promoting bifrost images: https://github.com/stackhpc/stackhpc-release-train/actions/runs/4224706743
